test/AreaMasterTest.cpp

```diff
@@ -0,0 +1,58 @@
+/**
+ * @file AreaMasterTest.cpp
+ * @brief AreaMasterクラスのテスト
+ * @author keiya121
+ */
+
+#include "AreaMaster.h"
+#include "Logger.h"
+#include <gtest/gtest.h>
+#include <gtest/internal/gtest-port.h>
+
+using namespace std;
+
+namespace etrobocon2024_test {
+
+  // 左コースで指定動作を行う場合のテスト
+  TEST(AreaMasterTest, runLeftCourse)
+  {
+    Area area = Area::AreaMaster;
+    bool isLeftCourse = true;
+    bool isLeftEdge = isLeftCourse;
+    int targetBrightness = 45;
+
+    testing::internal::CaptureStdout();  // 標準出力キャプチャ開始
+    class AreaMaster areaMaster(area, isLeftCourse, isLeftEdge, targetBrightness);
+    areaMaster.run();
+    string output = testing::internal::GetCapturedStdout();  // キャプチャ終了
+
+    // find("str")はstrが見つからない場合string::nposを返すß
```

Comment:
ß ← エスツェット文字っていうのが入ってたから、一応気になったので修正お願いします。
```suggestion
    // find("str")はstrが見つからない場合string::nposを返す
```

Reply:
ご確認いただきありがとうございます。修正しました。